src/services/WolfDen.Domain/Entity/AttendenceClose.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace WolfDen.Domain.Entity
+{
+    public class AttendenceClose
+    {
+        public int Id { get;private set; }
+        public int Month { get;private set; }

```

Comment:
Should int suffice

Reply:
A year column too can be added

Reply:
You can just use a date field to represent the same , like for every month it will have a date less than those date attendance data can't be edited